container/containerd/client.go

```diff
@@ -114,6 +115,9 @@ func (c *client) TaskPid(ctx context.Context, id string) (uint32, error) {
 	if err != nil {
 		return 0, errdefs.FromGRPC(err)
 	}
+	if response.Process.Status == tasktypes.StatusUnknown {
```

Comment:
with this we are not getting into the fast backoff retry mechanism, we are failing to create a handler and re-attempt to create it at the next cycle. WOuld it be better to first retry in the small cycle inside the handler creation logic?

Reply:
@SergeyKanzhelev return `ErrUnknown` here will end up with a retry in the loop here:

https://github.com/kubernetes/kubernetes/blob/master/vendor/github.com/google/cadvisor/container/containerd/handler.go#L96-L110

it loops there at a max of 5 times. before this PR we would not have looped at all.

Reply:
That's a great point. Should we consider to remove the `!errdefs.IsNotFound(err)` check from https://github.com/kubernetes/kubernetes/blob/master/vendor/github.com/google/cadvisor/container/containerd/handler.go#L105 and retry on _all_ errors?

Reply:
I think this is the best course of action. Since the retry period is small, this should be fine.